_posts/2020-02-27-react-conf-2020.markdown

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title:  "ReactConf 2020"
+date:   2020-03-02
+tags: React Sydney
+author: Natalie Laing
+image: /img/blog/reactConf/react.png
+---
+
+## Introduction 
+React Conf AU is the first React conference in the southern hemisphere and it didn't disappoint. The quality of speakers was spot on, meet the faces behind the projects that make React what it is today.
+
+## Day 1
+![day 1](/img/blog/reactConf/conf_day1.jpg)
+
+### Keynote css in js
+Max Stoiber (Gatsby) 
+
+
+The father of styled components took us through CSS in js a history.
```

Comment:
> took us through CSS in js a history.

Doesn't make sense?

Reply:
Sorry, Harry Potter reference. I will muggle proof it.